tests/src/common/Wsk.scala

```diff
@@ -276,7 +276,13 @@ class WskAction()
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
         val params = Seq(noun, if (!update) "create" else "update", "--auth", wp.authKey, fqn(name)) ++
-            { artifact map { Seq(_) } getOrElse Seq() } ++
+            // split artifact, apply name resolution and reconstruct back the string
+            {
+                kind map {
+                    case "sequence" => artifact map { art => Seq(art.split(",").map(fqn(_)).mkString(",")) } getOrElse Seq()
+                    case _ => artifact map { Seq(_) } getOrElse Seq()
+                } getOrElse { artifact map { Seq(_) } getOrElse Seq() }
+            } ++
```

Comment:
You got quite a few repetitions of `getOrElse Seq()` and so on in there. You can abuse `Option`s `Seq`-like behaviour (either one or none element) to to write the whole thing a bit more concise and understandable.

**Note:** The old code is equivalent to just `{ artifact }`. There is no need to convert the `Option` to a `Seq` (for which a `toSeq` would've been enough as well). You can always write `Seq() ++ someOption` because `Option` is "iterable" itself. `Option` can be read as a List of either 1 or 0 entries. Thus: `Seq() ++ someOption` will produce an empty list, if the `Option` is `None`, or a list of exactly 1 entry... you get the idea.

That being said, the code above can be refactored to:

```
kind match {
    case Some("sequence") => artifact.map(_.split(",").map(fqn(_)).mkString(","))
    case _ => artifact
}